cmd/cmd.go

```diff
@@ -20,13 +20,13 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"slices"
```

Comment:
I think we need to upgrade to go 1.21 to use slices and cmp

Reply:
@JeffFaer @katexochen I think this broke `GitHub Actions` + `Pre-commit` as I need to set up Go with newer version `1.21.x`
```yml
      - name: Setup Go
        uses: actions/setup-go@6c1fd22b67f7a7c42ad9a45c0f4197434035e429 # v5.0.0
        with:
          go-version: '1.21.x'
```
 Else i get the the following error.
```log
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/go', 'install', './...')
return code: 1
stdout: (none)
stderr:
    go: downloading github.com/spf13/pflag v1.0.5
    go: downloading golang.org/x/exp v0.0.0-20240103183307-be819d1f06fc
    go: downloading github.com/Workiva/go-datastructures v1.0.53
    go: downloading github.com/rs/zerolog v1.31.0
    go: downloading github.com/mattn/go-isatty v0.0.20
    go: downloading golang.org/x/sys v0.15.0
    go: downloading github.com/mattn/go-colorable v0.1.13
    keepsorted/block.go:18:2: package cmp is not in GOROOT (/opt/hostedtoolcache/go/1.20.13/x64/src/cmp)
    note: imported by a module that requires go 1.21
    keepsorted/block.go:20:2: package slices is not in GOROOT (/opt/hostedtoolcache/go/1.20.13/x64/src/slices)
    note: imported by a module that requires go 1.21
```

Previously I was not it did not require the set-up (GitHub Actions default `golang` has default ?).
Could you please update the release notes to reflect this? Or something was not set up correctly on our end?  
Old GitHubAction yml - https://github.com/GoogleChromeLabs/chromium-bidi/blob/455a1e720ab96c4d98a77417bd45f9d64a457c67/.github/workflows/pre-commit.yml
PR with update -
https://github.com/GoogleChromeLabs/chromium-bidi/pull/1829/files#diff-ac5eead3f3ce4863c524fff031a87b7aecccb4a0493df087a4e1c704a1505036R27

Reply:
I'd say ideally we would also add pre-built binaries to the releases, so we don't require Go at all.

Reply:
I've had another person internally ask for pre-built binaries, so I'm not entirely opposed...but how does that play with pre-commit? I guess maybe we'd turn the pre-commit hook into a script that fetches the pre-built binary and executes it?

----

https://pre-commit.com/#golang
> new in 3.0.0: pre-commit will bootstrap go if it is not present. language: golang also now supports language_version

I'm gunna check to see if that will fix the problem without forcing everyone to update their github actions

Reply:
1. It looks like `language_version` bootstrapping should work: https://github.com/google/keep-sorted/pull/21
2. pre-commit native support for pre-built binaries is also in the works: https://github.com/pre-commit/pre-commit/issues/1453

Reply:
This should be fixed in [v0.3.1](https://github.com/google/keep-sorted/releases/tag/v0.3.1). Let me know if you find yourself still needing to install go 1.21 separately from pre-commit.